client/src/event/Auth.vue

```diff
@@ -79,5 +82,10 @@
         }
       },
     },
+    created() {
+      if (this.$route.query.authFailed) {
```

Comment:
Why not just use this in the `v-if`? Seems weird to create a "fake" error.

Reply:
I'd avoid using vue internal states in the front-end

Reply:
I see no problem in using `$route` in the template personally. Either way though, I don't like the crafting of an error, especially since that error is usually the result of the form submission.